File: 0242-valid-anagram/0242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        if(s.size()!= t.size()) return false;

        int hashmap[26] = {0};
        for( char  ch : s){
            hashmap[ch-'a']++;
        }

        for( char  ch : t){
            int ind = ch-'a';

            if(hashmap[ind]==0){
                return false;
            }else{
                hashmap[ind]--;
            }
        }
        return true;
    }
};

File: 0242-valid-anagram/0242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {

        if(s.length() != t.length())
            return false;
        int[] hashmap = new int[26];

        for (char ch : s.toCharArray()) {
            hashmap[ch - 'a']++;
        }

        for (char ch : t.toCharArray()) {
            if (hashmap[ch - 'a'] == 0) {
                return false;
            } else {
                hashmap[ch - 'a']--;
            }
        }
        return true;
    }
}